plugins/nanospell/plugin.js

```diff
@@ -812,9 +829,8 @@
 					word;
 
 				range.selectNodeContents(block);
-				var bookmarks = editor.getSelection().createBookmarks(true);
```

Comment:
Basicalyl "startCheckOrMarkWords" eventually either calls this `getWords` function